params/aa_loxodromic.json

```diff
@@ -6,16 +6,19 @@
     "plotter": {
```

Comment:
This one looks super cool now, consider making a screenshot/gif for the gallery.